lib/main.dart

```diff
@@ -60,6 +62,15 @@ class _MyHomePageState extends State<MyHomePage> with TickerProviderStateMixin {
     @required this.store,
   });
 
+  Future<Null> getLocation() async {
+    LocationService location = new LocationService();
+    try {
+      currentLocation = await location.getLocation;
+    } on PlatformException {
+      print(MissingPluginException);
```

Comment:
I don't think this will work, it looks like it's printing a type instead of an exception instance.

It should look like this:

```
} on PlatformException catch(e) {
  print(e);
}
```